src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -514,14 +533,14 @@ export class NetworkRequest {
       this.#response.extraInfo = undefined;
     }
 
-    // TODO: get headers from Fetch.requestPaused
-    const headers = bidiNetworkHeadersFromCdpNetworkHeaders(
-      this.#response.info?.headers
-    );
+    const headers = [
+      ...bidiNetworkHeadersFromCdpNetworkHeaderEntries(
+        this.#response.paused?.responseHeaders
```

Comment:
I wonder why no e2e is failing after this change

Reply:
We never enable interception for most test. Naturally the event will not be present.